tr-site/src/assets/js/freelancer.js

```diff
@@ -0,0 +1,44 @@
+// Freelancer Theme JavaScript
```

Comment:
You're adding freelancer.js and freelancer.min.js we can remove both and have that small logic in another place of our current project, if you see this complicated, reduce the logic of the file with the minimun functions and keep the min.js